src/components/ciConfig/CIBuildpackBuildOptions.tsx

```diff
@@ -0,0 +1,627 @@
+import React, { useEffect, useState } from 'react'
+import ReactSelect, { components } from 'react-select'
+import {
+    DropdownIndicator,
+    getCommonSelectStyle,
+    getCustomOptionSelectionStyle,
+    Option,
+    OptionWithIcon,
+    ValueContainerWithIcon,
+} from '../v2/common/ReactSelect.utils'
+import { ReactComponent as GitLab } from '../../assets/icons/git/gitlab.svg'
+import { ReactComponent as Git } from '../../assets/icons/git/git.svg'
+import { ReactComponent as GitHub } from '../../assets/icons/git/github.svg'
+import { ReactComponent as BitBucket } from '../../assets/icons/git/bitbucket.svg'
+import { ReactComponent as QuestionIcon } from '../v2/assets/icons/ic-question.svg'
+import { ReactComponent as HelpIcon } from '../../assets/icons/ic-help.svg'
+import { getAbsoluteProjectPath, _multiSelectStyles } from './CIConfig.utils'
+import { OptionType } from '../app/types'
+import { CIBuildType, DockerConfigOverrideKeys } from '../ciPipeline/types'
+import {
+    BuilderIdOptionType,
+    CIBuildpackBuildOptionsProps,
+    InitLanguageOptionType,
+    LanguageBuilderOptionType,
+    LanguageOptionType,
+    VersionsOptionType,
+} from './types'
+import TippyWhite from '../common/TippyWhite'
+
+export const renderOptionIcon = (option: string) => {
+    if (!option) return null
+
+    const optionIconAvailable = option.includes('gitlab') || option.includes('github') || option.includes('bitbucket')
```

Comment:
We have constant for `github` `gitlab` `bitbucket` but in sso login. Can we make it common and use from there